src/core/Note.cpp

```diff
@@ -74,7 +74,8 @@ Note::Note( const Note & note ) :
 	m_panning( note.m_panning ),
 	m_length( note.m_length ),
 	m_pos( note.m_pos ),
-	m_detuning( nullptr )
+	m_detuning(nullptr),
```

Comment:
The formatting change of this line can be undone and the memberwise initialization of the following ``m_type(note.m_type)`` can stay.

Reply:
I kept the formatting since I'd have to change this line anyways on the PR (by adding the comma)